build.sh

```diff
@@ -2,14 +2,30 @@
 
 set -euxfo pipefail;
 
+source "_common.sh";
+
 build () {
   declare -r dockerfile_name="${1}";
   declare -r base_image="${2}";
   declare -r installer_url="${3}";
   declare -r version="${4}";
+  declare -r variant="${5}";
+
+  declare tag="${USERNAME}/racket:${version}"
```

Comment:
Instead of adding a `variant` arg and inferring the tag from `variant` and `version`, what do you think about passing the tag directly? I'd prefer to avoid adding the complexity of the case statement.

Reply:
Will do!